client/state/billing-transactions/reducer.js

```diff
@@ -46,6 +49,48 @@ export const requesting = createReducer( false, {
 	[ BILLING_TRANSACTIONS_REQUEST_SUCCESS ]: () => false,
 } );
 
+/**
+ * Returns an updated subtree for individual receipts that are not found in the receipt list
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Action payload
+ * @return {Object}        Updated state
+ */
+export const individualTransactions = createReducer(
+	{},
+	{
+		[ BILLING_RECEIPT_REQUEST ]: ( state, { receiptId } ) => ( {
+			...state,
+			requesting: {
```

Comment:
In the original reducer for all billing transactions we have separate reducers for tracking request state and storing transaction data. Seems like we use the same reducer for that here. 

My personal opinion is that if we keep reducers separate, it would allow us to use `keyedReducer` and keep the code with more separate concerns, and arguably more consistent and easier to test.

Would love to hear your thoughts about this.